pages/_app.tsx

```diff
@@ -45,7 +43,6 @@ export default function Layout({
         <title>BitcoinBeach Lightning Node</title>
       </Head>
       <GraphQLProvider>
-        <Header />
         <Component {...pageProps} />
```

Comment:
Same warning here. The app still works with the warning though `'Component' cannot be used as a JSX component` @daviroo @samerbuna